cset-workflow/rose-suite-metoffice_lfric.conf

```
@@ -0,0 +1,55 @@
+[template variables]
+!!CAPE_RATIO_TIME=""
+CLEAN_WEB_DIR=True
+!!CONDA_METPLUS_VENV_LOCATION=""
+CONDA_PATH=""
+CONDA_VENV_CREATE=True
+CONDA_VENV_LOCATION="$DATADIR/CSET/workflow-conda-env"
+CSET_CYCLE_PERIOD="PT1H"
+CSET_ENV_SEPARATE_MET=False
+CSET_ENV_USE_CONDA=True
+CSET_ENV_USE_LOCAL_CSET=True
+CSET_ENV_USE_MODULES=False
+!!CSET_FILE_NAME_METADATA_PATTERN=""
+!!CSET_FILE_TIME_OFFSET=0
+CSET_FINAL_CYCLE_POINT="2019-07-27T00:00Z"
```

```suggestion
CSET_FINAL_CYCLE_POINT=""
```
Remove values that can't be the default.